tests/python/test_moe.py

```diff
@@ -104,10 +112,8 @@ def run_routed_experts(self, hidden_states: torch.Tensor) -> torch.Tensor:
         router_scores = topk_weight.sigmoid()  # [s, 1]
         hidden_states = hidden_states * router_scores  # [s, h]
 
-        counts = topk_ids.new_zeros(
-            (topk_ids.size(0), self.config.num_routed_experts)
-        )  # [s, n]
-        counts.scatter_(1, topk_ids, 1)  # [s, n]
+        counts = torch.zeros(topk_ids.size(0), self.config.num_routed_experts, device=topk_ids.device, dtype=torch.int32)  # [s, n]
```

Comment:
Using `torch.zeros` as for `new_zeros`, thunder falls back to inductor at the moment.